src/promptflow/promptflow/_cli/data/package_tool/setup.py.jinja2

```diff
@@ -0,0 +1,17 @@
+from setuptools import find_packages, setup
+
+PACKAGE_NAME = "{{ package_name }}"
+
+setup(
+    name=PACKAGE_NAME,
+    version="0.0.1",
```

Comment:
so when user use this CLI, the version will always be `0.0.1`? will we support custom this?